Documentation/doc/Documentation/Developer_manual/create_and_use_a_cmakelist.txt

```diff
@@ -0,0 +1,74 @@
+/*!
+\page devman_create_and_use_a_cmakelist How to use CGAL with CMake
```

Comment:
```suggestion
\page devman_create_and_use_a_cmakelist How to use %CGAL with CMake
```
... to remove the automatic linking of `CGAL`.

Reply:
@sloriot The initial issue (a link on "CGAL") is still there, now that you can reverted to using `CGAL`. I think it should be `%CGAL`.

See here: https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.1-Ic-127/output2/Manual/devman_create_and_use_a_cmakelist.html

Reply:
I saw yesterday that it was breaking the navtree but actually it was another line. Fixed in 2dc7f66c8a5.